concepts/conditionals-if/.meta/config.json

```diff
@@ -1,5 +1,5 @@
 {
-  "blurb": "TODO: add blurb",
+  "blurb": "If statements allow to control the flow of execution of a program based on a boolean condition.",
```

Comment:
Grammar improvement:

If statements control the flow of execution of a program based on a boolean condition.